ebuild-writing/common-mistakes/text.xml

```diff
@@ -400,12 +411,14 @@ SLOT="0"
 <body>
 
 <p>
+Make sure the <c>DESCRIPTION</c> is not overly long. Good descriptions will
+describe the main function of the package in a sentence.
```

Comment:
I personally wouldn't add this, since it's covered by repoman and pkgcheck already. Can't say this has been a **common** mistake we've seen.

Reply:
This is just reformatting from [`7ef76ee` (#223)](https://github.com/gentoo/devmanual/pull/223/commits/7ef76ee3e006e2d0e92fe81cfa8f15616db59494)